token/confidential-transfer/proof-generation/Cargo.toml

```diff
@@ -8,11 +8,14 @@ license = "Apache-2.0"
 edition = "2021"
 
 [dependencies]
-curve25519-dalek = "3.2.1"
-solana-zk-sdk = "2.0.3"
+curve25519-dalek = "4.1.3"
```

Comment:
Thanks for updating this. It would be great to get a new release of `spl-token-confidential-transfer-proof-generation` with these changes included because it's currently conflicting with `solana-* 2.1.0`:

```
error: failed to select a version for `zeroize`.
    ... required by package `curve25519-dalek v3.2.1`
    ... which satisfies dependency `curve25519-dalek = "^3.2.1"` of package `spl-token-confidential-transfer-proof-generation v0.1.0`
    ... which satisfies dependency `spl-token-confidential-transfer-proof-generation = "^0.1.0"` of package `spl-token-2022 v5.0.2`
    ... which satisfies dependency `spl-token-2022 = "^5.0.2"` of package `native v0.1.0 (/home/x/native")`
versions that meet the requirements `>=1, <1.4` are: 1.3.0, 1.2.0, 1.1.1, 1.1.0, 1.0.0

all possible versions conflict with previously selected packages.

  previously selected package `zeroize v1.7.0`
    ... which satisfies dependency `zeroize = "^1.7"` of package `solana-zk-sdk v2.1.0`
    ... which satisfies dependency `solana-zk-sdk = "^2.1.0"` of package `native v0.1.0 (/home/x/native")`

failed to select a version for `zeroize` which could resolve this conflict
```

Reply:
Yeah... this is going to be another one of those painful upgrades. I'll publish all the required SPL crates today and tomorrow so we can backport to Solana v2.1